app/dashboard/migrations/0110_auto_20200517_0514.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.4 on 2020-05-16 13:19
+
+from django.db import migrations, models
+
+def populateFunderAddress(apps, schema_editor):
+    BountyFulfillment = apps.get_model('dashboard', 'BountyFulfillment')
+
+    for fulfillment in BountyFulfillment.objects.all():
+        fulfillment.funder_address = fulfillment.bounty.bounty_owner_address
```

Comment:
nice! Can we backfill some of the other fields as well?

Reply:
like ? I'm not sure which one else we would need to backfill -> 
I've got a follow up PR which retires a few fields which don't have data 

@danlipert

Reply:
I'll have that PR opened up today ! Just testing out flow